Sources/Build/PkgConfig.swift

```diff
@@ -16,32 +16,52 @@ enum PkgConfigError: ErrorProtocol {
 }
 
 struct PkgConfig {
-    static let searchPaths = ["/usr/local/lib/pkgconfig",
+    private static let searchPaths = ["/usr/local/lib/pkgconfig",
                               "/usr/local/share/pkgconfig",
                               "/usr/lib/pkgconfig",
                               "/usr/share/pkgconfig",
                               ]
     
     let name: String
     let pcFile: String
-    var cFlags = [String]()
-    var libs = [String]()
-    private var parser: PkgConfigParser
+    let cFlags: [String]
+    let libs: [String]
     
     init(name: String) throws {
         self.name = name
         self.pcFile = try PkgConfig.locatePCFile(name: name)
-        parser = PkgConfigParser(pcFile: pcFile)
+        
+        var parser = PkgConfigParser(pcFile: pcFile)
+        try parser.parse()
+        
+        var cFlags = [String]()
+        var libs = [String]()
+        
+        // FIXME: handle spaces in paths.
```

Comment:
Let's make sure we have a bug in the tracker for this, I don't want it to get lost.